app/presenters/sample_presenter.rb

```diff
@@ -1,13 +1,14 @@
 class SamplePresenter
   delegate_missing_to :@sample
 
-  def self.map(samples, format)
-    samples.map { |sample| new(sample, format) }
+  def self.map(samples, format, unblind= false)
```

Comment:
I would make it keyword argument: 

```suggestion
  def self.map(samples, format, unblind: false)
```